documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -112,7 +113,11 @@ You can migrate virtual machines to {virt} from the command line.
 
 [IMPORTANT]
 ====
-You must ensure that all xref:prerequisites[prerequisites] are met.
+*  You must be logged in as a user with the {ocp} `cluster-admin` privileges.
```

Comment:
@sbeskin-redhat I think the 'the' is irrelevant here, you don't point on specific rules like for VMware

Reply:
+1